LICENSE.md

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 GitHub
+Copyright (c) 2021 GitHub
```

Comment:
IANAL, but if the [answer on this post](https://opensource.stackexchange.com/questions/5778/why-do-licenses-such-as-the-mit-license-specify-a-single-year) is accurate, this should probably be `2018-2021`

Reply:
Not only that, but

> For software with several releases over multiple years, it's okay to use a range (“2008-2010”) instead of listing individual years (“2008, 2009, 2010”) if and only if every year in the range, inclusive, really is a “copyrightable” year that would be listed individually; and you make an explicit statement in your documentation about this usage.

I think we might have skipped some years in there.

As far as I can tell, it should be pretty acceptable to leave this at "2018" and not have to do this dance every year. I _think_ that the only effect of adding later years would be to make it clear to the reader that the copyright will still be in effect until `2021+N` rather than `2018+N`, where `N` is a sufficiently large number that I can't imagine it will make a difference.

Reply:
Seems reasonable to me. 👍🏼